packages/renderer/src/lib/troubleshooting/TroubleshootingGatherLogs.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+import { faDownload, faFileLines, faScroll } from '@fortawesome/free-solid-svg-icons';
+import type { FileMap } from '../../../../main/src/plugin/troubleshooting';
+import Button from '../ui/Button.svelte';
+import Fa from 'svelte-fa';
+import { generateLogFileName } from '../../../../main/src/plugin/troubleshooting';
+
+let logs: FileMap[] = [];
+let hasLogs = false;
+
+let fetchLogsInProgress = false;
+
+async function retrieveSingleLog(getLog: () => Promise<FileMap[]>): Promise<void> {
+  try {
+    const log = await getLog();
+    logs = [...logs, ...log];
+  } catch (error) {
+    console.error('Error retrieving log:', error);
+  }
+}
+
+// Retrieve all the logs asynchronously
+async function retrieveLogs() {
+  fetchLogsInProgress = true;
+
+  // "Reset" the logs array
+  logs = [];
+
+  // Retrieve all the logs and update the fetch progress status
+  const logFunctions = [window.troubleshootingGetConsoleLogs, window.troubleshootingGetSystemLogs];
+  await Promise.all(logFunctions.map(func => retrieveSingleLog(func)));
+
+  fetchLogsInProgress = false;
+}
+
+// Save files as a zip file (we first ask the user for the dialog, and then save the files to the filepath)
+async function saveLogsAsZip() {
+  const result = await window.saveFileDialog('Save Logs as .zip', generateLogFileName('podman-desktop') + '.zip');
+  if (!result.canceled && result.filePath) {
+    await window.saveLogsToZip(logs, result.filePath);
+  }
+}
+
+// Added so we can reactively disable / enable the download button.
+$: hasLogs = logs.length > 0;
+</script>
+
+<div class="flex flex-col w-full m-4 bg-charcoal-600 p-4 rounded-lg">
+  <div class="flex flex-row align-middle items-center w-full">
+    <Fa size="30" class="pr-3 text-gray-700" icon="{faFileLines}" />
+    <div class="text-xl">Gather Log Files</div>
+    <div class="flex flex-1 justify-end"></div>
+  </div>
+  <div class="mt-4">Bundle all available logs into a .zip</div>
+  <div class="mt-4">
+    <Button
+      inProgress="{fetchLogsInProgress}"
+      on:click="{() => retrieveLogs()}"
+      title="Collect logs for must gather tool"
+      icon="{faScroll}">Collect logs</Button>
+    <Button on:click="{() => saveLogsAsZip()}" title="Download .zip file" icon="{faDownload}" disabled="{!hasLogs}"
```

Comment:
I feel like, this is redundant to click on collect, then click on save as zip.
We should be able to quickly do it in a single click

Reply:
Similar feedback to what I was saying earlier, so I agree it should just be one button.

Maybe we can drop the 'console output' then too - if you care what files are included you can just open the zip before you give it to anyone.

With the new feedback tabs it also feels a little out of place as it's own tab (maybe should be on Logs tab? or maybe there's a different way to organize them) but we can always reorganize things later.